qa/TL1_custom_cpp_lib/test.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash -e
+
+test_body() {
+    mkdir -p /tmp/custom_cpp_lib
+    git clone https://github.com/NVIDIA/DALI /tmp/dali
+    pushd /tmp/custom_cpp_lib
+    echo "
+#include <dali/c_api.h>
+int main() {
+  daliInitialize();
+  daliInitOperators();
+}
+" > main_stub.cc
+    echo "
+cmake_minimum_required(VERSION 3.13)
+project(CustomCppLib)
+include(/tmp/dali/tools/find_dali.cmake)
+add_library(test_lib SHARED main_stub.cc)
+find_dali(DALI_INCLUDE_DIR DALI_LIB_DIR DALI_LIBRARIES)
+target_include_directories(test_lib ${DALI_INCLUDE_DIR})
+target_link_directories(test_lib ${DALI_LIB_DIR})
+target_link_libraries(test_lib ${DALI_LIBRARIES})
+" > CMakeLists.txt
+    mkdir build
+    pushd build
+    cmake -D CUDA_TARGET_ARCHS=60 -D CMAKE_BUILD_TYPE=Debug ..
```

Comment:
```suggestion
    cmake -D CUDA_TARGET_ARCHS=60  ..
```
Do we care that it is a debug build?

Reply:
You are not building anything GPU specific, so why make it more complicated it need to be.

Reply:
Removed